app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe):
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
+    else:
+        return f"Friends can go to {cafe.name}"
```

Comment:
you can avoid else with return statement

Reply:
ok